DeviceSettings/res/values-zh-rCN/aosip_strings.xml

```diff
@@ -11,106 +11,119 @@
-    <string name="touchscreen_gesture_one_finger_up_swipe_title">单指向上滑动</string>
-    <string name="touchscreen_gesture_one_finger_up_swipe_home_title">从主页键向上滑动</string>
-    <string name="touchscreen_gesture_one_finger_down_swipe_title">单指下滑</string>
-    <string name="touchscreen_gesture_one_finger_left_swipe_title">单指向左滑动</string>
-    <string name="touchscreen_gesture_one_finger_right_swipe_title">单指向右滑动</string>
-    <string name="touchscreen_gesture_single_tap_title">单击</string>
-    <string name="touchscreen_gesture_down_arrow_title">绘制一个\"Λ\"</string>
-    <string name="touchscreen_gesture_up_arrow_title">绘制一个 “V”</string>
-    <string name="touchscreen_gesture_right_arrow_title">绘制一个\"&lt;\"</string>
-    <string name="touchscreen_gesture_left_arrow_title">绘制一个\"&gt;\"</string>
-    <string name="touchscreen_gesture_letter_c_title">画个字母“C”咯</string>
-    <string name="touchscreen_gesture_letter_e_title">绘制字母\"e\"</string>
-    <string name="touchscreen_gesture_letter_o_title">绘制字母“O”</string>
-    <string name="touchscreen_gesture_letter_m_title">绘制字母\"M\"</string>
-    <string name="touchscreen_gesture_letter_s_title">绘制字母\"S\"</string>
-    <string name="touchscreen_gesture_letter_v_title">绘制字母\"V\"</string>
-    <string name="touchscreen_gesture_letter_w_title">绘制字母\"W\"</string>
-    <string name="touchscreen_gesture_letter_z_title">绘制字母 “Z”</string>
-    <!-- Screen off Gestures : Actions -->
-    <string name="touchscreen_gesture_action_dialog_title">手势操作</string>
-    <string name="touchscreen_gesture_action_do_nothing">什么都不做</string>
-    <string name="touchscreen_gesture_action_flashlight">切换手电筒</string>
-    <string name="touchscreen_gesture_action_camera">打开相机</string>
-    <string name="touchscreen_gesture_action_browser">打开浏览器</string>
-    <string name="touchscreen_gesture_action_dialer">打开拨号器</string>
-    <string name="touchscreen_gesture_action_email">查看您的电子邮件</string>
-    <string name="touchscreen_gesture_action_messages">查看您的信息</string>
-    <string name="touchscreen_gesture_action_play_pause_music">播放/暂停音乐</string>
-    <string name="touchscreen_gesture_action_previous_track">切换到前一曲目</string>
-    <string name="touchscreen_gesture_action_next_track">切换到下一曲目</string>
-    <string name="touchscreen_gesture_action_volume_down">降低媒体音量</string>
-    <string name="touchscreen_gesture_action_volume_up">提高媒体音量</string>
-    <string name="touchscreen_gesture_action_ambient_display">环境显示</string>
-    <string name="touchscreen_gesture_action_wake_device">唤醒设备</string>
+     <!-- Screen off Gestures : Input -->
+     <string name="touchscreen_gesture_settings_title">屏幕手势</string>
+     <string name="touchscreen_gesture_settings_summary">为快捷操作采用各种手势</string>
+     <string name="touchscreen_gestures_haptic_feedback_title">手势反馈</string>
+     <string name="touchscreen_gestures_haptic_feedback_summary">检测到手势时振动</string>
+     <string name="touchscreen_gesture_two_finger_down_swipe_title">双指下划</string>
```

Comment:
this string should now function as intended.
see: https://github.com/yaap/device_oneplus_sm8150-common/commit/da51d21872861e0eda67e058e2c3ee63a462d22e